components/osgi-tests/src/test/java/org/wso2/carbon/analytics/test/osgi/SiddhiEditorTestCase.java

```diff
@@ -266,7 +319,7 @@ public void testCheckingWhetherInvalidFileExistsInWorkspace() throws Exception {
     }
 
     @Test
-    public void testFailureInExportingAFile() throws Exception {
+    public void testExportingAFile() throws Exception {
```

Comment:
Why are we changing a negative testcase to positive. Aren't we testing the negative behavior?

Reply:
Negative scenario has been already tested in [this](https://github.com/minudika/carbon-analytics/blob/b6c1df854a4577a74c82aa4e7150bc61c93f3506/components/osgi-tests/src/test/java/org/wso2/carbon/analytics/test/osgi/SiddhiEditorTestCase.java#L352) method.